docs/quick-start.md

```diff
@@ -0,0 +1,110 @@
+# DAG Factory
+
+**DAG Factory** is a library for [Apache Airflow®](https://airflow.apache.org) that allows you to construct DAGs declaratively via configuration files.
+
+# Prerequisites
+
+The minimum requirements for **dag-factory** are:
+
+- Python 3.8.0+
+- [Apache Airflow®](https://airflow.apache.org) 2.0+
+
+
+> **Note:** We will use the [astro-cli](https://www.astronomer.io/docs/astro/cli/overview) tool to set up the Airflow environment.
+
+
+# Initialize Apache Airflow Project
+
+To initialize your [Apache Airflow®](https://airflow.apache.org) project using astro-cli, follow these steps:
```

Comment:
I have mixed feelings about us using `astro-cli` in the project quickstart.

Since it's an open-source project agnostic to Astro, I believe we should have a quick start with plain Airflow, and we could have a dedicated page for running in Astro that uses Astro CLI, WDYT?

We had done this for Astro SDK in the past after a few discussions:
https://github.com/astronomer/astro-sdk?tab=readme-ov-file#quickstart

Reply:
make sense, I have changed to use Airflow standalone.